scipy/spatial/__init__.py

```diff
@@ -5,6 +5,15 @@
 
 .. currentmodule:: scipy.spatial
 
+Spatial Transformations
+=======================
+Contained in the :mod:`scipy.spatial.transform` submodule.
+
+.. autosummary::
```

Comment:
I suggest to eliminate `.. autosummary::` section here, because it is displayed in a weird way. And it doesn't seem necessary. Just a link to the dedicated `spatial.transform` page will be sufficient.

Reply:
I tried to eliminate that bug for hours until I saw the same thing on the `scipy.sparse` documentation page. I'll make the changes.

Reply:
I don't think it is a bug per se, it is supposed to extract some summary automatically. Just remove the section and it's good.